test/e2e/contract-test/index.html

```diff
@@ -1,51 +1,64 @@
 <html>
 <head>
+  <meta charset="UTF-8">
   <title>E2E Test Dapp</title>
+  <script src="node_modules/metamask-onboarding/dist/metamask-onboarding.bundle.js"></script>
```

Comment:
Does this need to be updated with the new URL?

```suggestion
  <script src="node_modules/@metamask/onboarding/dist/metamask-onboarding.bundle.js"></script>
```

Reply:
Yes, it does! Good catch.

Reply:
This has been fixed